bindata/v4.1.0/kube-controller-manager/pod.yaml

```diff
@@ -22,6 +22,23 @@ spec:
         sleep 1
       done
   containers:
+  - name: cluster-policy-controller
+    image: ${CLUSTER_POLICY_CONTROLLER_IMAGE}
+    imagePullPolicy: IfNotPresent
+    terminationMessagePolicy: FallbackToLogsOnError
+    command: ["cluster-policy-controller", "start"]
+    args:
+    - --config=/etc/kubernetes/static-pod-resources/configmaps/config/config.yaml
```

Comment:
I wasn't expecting a config

Reply:
removed, added kubeconfig

Reply:
it already merged

Reply:
yes, the container needs the config, w/out it I'm seeing (w/ local test, running the container manually in a test cluster) 
```
I1015 01:33:50.754554       1 policy_controller.go:41] Starting controllers on 0.0.0.0:8443 (v0.0.0-unknown)
F1015 01:33:50.754639       1 cmd.go:57] open /var/run/configmaps/client-ca/ca-bundle.crt: no such file or directory
```
I'm hoping somebody who has more experience with these controllers running in openshift-controller-manager pod can take a look, as I'm semi-unfamiliar w/ the actual controller code (although getting more familiar now)